tests/brevitas_examples/test_llm.py

```diff
@@ -404,28 +467,39 @@ def test_small_models_acc_pt_ge_2_4(caplog, acc_args_and_acc_pt_ge_2_4):
                 "L__self___model_layers_0_self_attn_k_proj":
                     "<class 'torch.nn.modules.linear.Linear'>",
                 "L__self___model_layers_0_self_attn_o_proj":
-                    "<class 'torch.nn.modules.linear.Linear'>"}},])
+                    "<class 'torch.nn.modules.linear.Linear'>"},
+            "exp_layer_types_count": {
+                "<class 'torch.nn.modules.linear.Linear'>":
+                    15,  # LM Head + Q/K/V projs + Up/Gate/Down projs
+                "<class 'torch.nn.modules.normalization.RMSNorm'>": 5,  # Input + Post attention
+                "<class 'torch.nn.modules.normalization.LayerNorm'>": 0,}},])
 def layer_args(default_run_args, request):
     args = default_run_args
     layer_dict = request.param
     exp_layer_types = layer_dict["exp_layer_types"]
+    exp_layer_types_count = layer_dict["exp_layer_types_count"]
     del layer_dict["exp_layer_types"]
+    del layer_dict["exp_layer_types_count"]
     args.update(**layer_dict)
-    yield args, exp_layer_types
+    yield args, exp_layer_types, exp_layer_types_count
 
 
-@pytest.mark.llm
-@requires_pt_ge('2.2')
 def test_small_models_quant_layer(caplog, layer_args):
     caplog.set_level(logging.INFO)
-    args, exp_layer_types = layer_args
+    args, exp_layer_types, exp_layer_types_count = layer_args
```

Comment:
My intuition is to split this into 2 different tests, 1 which checks the types of specific layer names, another which does the layer count tests. If we want the same models / configs tested then let's be verbose about that.